examples/basic/README.md

```diff
@@ -1,3 +1,8 @@
 # Deploy basic observability agents
 
 An end to end example using the module's default variable values.
+
+The example shows how to configure the module to setup the Logging agent for [Kubernetes metadata logs filtering](https://github.com/logdna/logdna-agent-v2/blob/3.8/docs/KUBERNETES.md#configuration-for-kubernetes-metadata-filtering) in order to:
+
+- include all log lines coming from the "default" Kubernetes namespace
+- exclude anything with a label `app.kubernetes.io/name` with value `sample-app` or an annotation `annotation.user` with value `sample-user`
```

Comment:
```suggestion
The example sets up the logging agent for [Kubernetes metadata filtering](https://github.com/logdna/logdna-agent-v2/blob/3.8/docs/KUBERNETES.md#configuration-for-kubernetes-metadata-filtering). 

The example configures the agent to include all log lines coming from the "default" Kubernetes namespace and exclude anything with a label `app.kubernetes.io/name` and value `sample-app` or an annotation `annotation.user` with the value `sample-user`.
```

words like `set up` / `setup` are confusing:
- 2 words if it's a verb: to set up the agent
- 1 words if it's a noun: the setup of the agent goes like this...

Reply:
thx, updated